CSS/sytlesheet.css

```diff
@@ -1,11 +1,33 @@
-/* media queries starts here*/
+/* media queries  here*/
 @media (min-width:1px) and (max-width: 550px) {
-    #logo {
-        border-radius: 5px;
-        width: 90px;
-    }
+#logo {
+border-radius: 5px;
+width: 90px;
+}
+#frasco {
```

Comment:
Just fix your indentation here please, apart from that, everything looks great 👍